docs/farming.md

```diff
@@ -221,6 +221,7 @@ services:
       "--state-pruning", "archive",
       "--port", "30333",
       "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
+      "--rpc-port", "9944",
```

Comment:
Why? It is the default port anyway.

Reply:
Can't we just set it explicitly for readability and avoid confusion with https://github.com/paritytech/substrate/pull/13384 which doesn't really say which port it actually runs on now?

Reply:
Does it matter to the user though? `"--node-rpc-url", "ws://node:9944",` is specified below, but user is not supposed to change this as there is no need for farming purposes.

Reply:
I have dropped the flag.